extensions/todoist/src/components/MenubarTask.tsx

```diff
@@ -99,6 +99,14 @@ const MenuBarTask = ({ task, data, setData }: MenuBarTaskProps) => {
   return (
     <View>
       <MenuBarExtra.Submenu title={removeMarkdown(task.content)} icon={getPriorityIcon(task)}>
+        <MenuBarExtra.Item
```

Comment:
What do you think of naming it `Open in Raycast` with the `Icon.RaycastLogoNeg` icon to make it clearer?

Reply:
sgtm! changing